### DISCUSSION TITLE
Roadmap toward 1.0.0 - your thoughts?

### DISCUSSION THREAD

**Discussion by probonopd** [helloSystem/hello#268] [General]
I have opened a ticket that will collect must-haves for 1.0.0.

Please feel free suggest must-haves for a 1.0.0 release. Once all must-haves are implemented, we may consider starting a test cycle toward a 1.0.0 release.

https://github.com/helloSystem/hello/issues/266

**Comment by louies0623** [comment]
We need a language setting that can change the system. This is the most basic human need, because not all computers have EFI that can automatically select the language correctly.
![image](https://user-images.githubusercontent.com/44593430/146654702-a701030c-9512-461e-a8ce-80d72a9c6839.png)

**Reply by louies0623** [reply]
The color of the startup screen Logo should be darker, like this.![image](https://user-images.githubusercontent.com/44593430/146655212-527ee3a8-284d-4a8a-b506-76e9eab217a9.png)

**Reply by 69graves** [reply]
Yes please!   Also of note, although it might be trivial, ttys are nearly unusable for someone with poor eyesight. And as far as I can tell, once cannot easily change this without digging into the kernel, is this correct?

**Reply by probonopd** [reply]
You can change `kern.vt.color.7.rgb="150,150,150"` to a darker color like `kern.vt.color.7.rgb="115,115,115"` in `/boot/loader.mute.d/loader.conf`.

**Reply by probonopd** [reply]
Added in https://github.com/helloSystem/ISO/commit/03b5dac7af5e68b1600e9198543b2980fc7f1315

**Comment by 69graves** [comment]
I think it would be quite helpful to have something of a hybrid between the more recent trend, in Linux with the "Welcome" screen that runs at first boot, or thereafter.   We already have the first boot "Hello" thing, so this seems to be something trivially implemented, and quite helpful to those who might not know their way around a FreeBSD system.   

This, and a keyboard shortcut overlay/cheat sheet would be interesting, perhaps invoked by pressing the 'Meta' key.

**Reply by probonopd** [reply]
> keyboard shortcut overlay/cheat sheet

Keyboard shortcuts are always printed in the menu next to the respective command. What would be the benefit of an extra sheet?

**Comment by alphamodh0** [comment]
An easy way to configure Bluetooth and connect peripherals of this type without having to know how to hack the matrix.

**Reply by alphamodh0** [reply]
Yesterday I tried it with my hearing aids and it was hell (note that in the end I did not succeed)

**Reply by probonopd** [reply]
Bluetooth is something that we should write a GUI utility for, or find an existing (Py)Qt one. Do you know a good one?

**Reply by alphamodh0** [reply]
Unfortunately no, yesterday I spent a good time searching and did not find anything like it. Almost everything is GTK.

**Reply by probonopd** [reply]
https://github.com/helloSystem/Menu/issues/46

**Comment by 69graves** [comment]
When I have the time later today I can do some looking.  Any special requirements other than no GTK?

**Reply by probonopd** [reply]
(Py)Qt, as few dependencies as possible, must not depend on "half of KDE Plasma".

**Comment by alphamodh0** [comment]
-A way to reinstall & upgrade without lose all my data or configurations.

-Add a recovery particion, like OSX, this would be very helpfully.
